src/test/SmartComponents/ImagesTable/ImagesTable.test.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import { renderWithReduxRouter } from '../../testUtils';
+
+import ImagesTable from '../../../SmartComponents/ImagesTable/ImagesTable';
+
+describe('ImagesTable Page', () => {
```

Comment:
from what I understand this isn't a stand-alone page

Reply:
I think it's fine to test components on their own no?

Reply:
It is, just the wording is misleading b/c this is a component, not a stand alone page.